climada/entity/impact_funcs/wildfire.py

```diff
@@ -0,0 +1,151 @@
+"""
+This file is part of CLIMADA.
+
+Copyright (C) 2017 ETH Zurich, CLIMADA contributors listed in AUTHORS.
+
+CLIMADA is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License as published by the Free
+Software Foundation, version 3.
+
+CLIMADA is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along
+with CLIMADA. If not, see <https://www.gnu.org/licenses/>.
+
+---
+
+Define impact functions for WildFires.
+"""
+
+__all__ = ['ImpfWildfire']
+
+import logging
+import numpy as np
+
+from climada.entity.impact_funcs.base import ImpactFunc
+
+LOGGER = logging.getLogger(__name__)
+
+class ImpfWildfire(ImpactFunc):
+    """Impact function for wildfire."""
+
+    def __init__(self):
+        ImpactFunc.__init__(self)
+        self.haz_type = 'WFsingle'
```

Comment:
The impact function is for single fires or for fire seasons? I thought the default for WildFire are seasons, hence I would expect the default Impact function to be for a season too...

Reply:
well the WildFire is both right? Thus we have the tag. So far I worked far more with WFsingle (for calibration and case studies) thus I thoguth I'd keep it. But happy to change

Reply:
The question is whether the impact function was calibrated for single fires or for fire seasons. There is a priori no reason to think that the two cases would lead to the same impact functions right?

Reply:
well, as I argue that MDD is constantly 100% and we calibrate PAA, it should be transferable under the assumption that within a season no centroid is hit (burned) more than once. Which is a fair assumption

Reply:
Ok, makes sense. I would then set the `haz_type` as an input argument to init. Probably it is also better then to NOT define a default, since this forces the user to be aware of whether the hazard is `WFsingle` or 'WFseason'.

Reply:
I've included it into the init and I'm raising a warning that informs about the hazard type, so that a user is better aware of it 42719ec8

I think it's helpful to keep a default, so that an impact calculation can run smoothly with the same logic as with any other hazard.